src/gwkokab/inference/bake.py

```
@@ -65,17 +65,27 @@ def __call__(self, **kwargs: Any) -> "Bake":
+    ) -> Tuple[
+        Dict[str, int | float | None],
+        Dict[str, Distribution],
+        Dict[str, str],
+        Callable[..., Distribution],
+    ]:
         """Return the distribution with the fixed parameters set.
 
         Returns
         -------
-        Tuple[Dict[str, Distribution], Dict[str, str], Callable[..., Distribution]]
+        Tuple[Dict[str, int | float | None], Dict[str, Distribution], Dict[str, str], Callable[..., Distribution]]
```

Pylint (3.3.7)

[convention] Line too long (114/100)

(C0301)